scripts/convert.py

```diff
@@ -40,6 +45,37 @@ def __init__(self, arguments):
         self.post_process = PostProcess(arguments)
         self.verify_output = False
 
+        ##initialize last_used face to whichever face is first
+        self.matched_face = 0
+        ##load images for target face id
+        try:
+            models_dir = self.args.model_dir
+            target_images = [face_recognition.load_image_file(file) for file in glob.glob(models_dir + '**\\*.png', recursive=True)]
+        except:
```

Comment:
Consider using cv2.imread and os.walk  with file type constaints
or  
```from scripts.fsmedia import Images``` class to load images

Reply:
using os.listdir and Images class - good tip, thanks